debian/rules.in

```diff
@@ -33,9 +35,10 @@ override_dh_auto_install:
 
 override_dh_shlibdeps:
 	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info
-	if [ -e debian/ucx-cuda.substvars ]; then \
-	  sed -i -e 's/libnvidia-compute-\([0-9]\+\)/libnvidia-compute | libnvidia-ml1/' \
-	    debian/ucx-cuda.substvars \
-	; fi
+	if [ -e $(CUDA_SUBSTVARS) ]; then \
+		sed -i -e 's/libnvidia-compute-\([0-9]\+\), //g' $(CUDA_SUBSTVARS); \
+		echo "shlibs:Recommends=libnvidia-compute, libnvidia-ml1" >> \
```

Comment:
is it ok that before the change it was , and now it is | ?

Reply:
So originally it depends on having either one of those installed. The new weak dependency (Recommends)  generated is for both of them to be installed. Looks different indeed.

Reply:
this is the intent to install both, @rakhmets pls correct me if im wrong

Reply:
Reverted.